src/main/resources/assets/computercraft/lang/de_DE.lang

```diff
@@ -0,0 +1,64 @@
+tile.computercraft:computer.name=Computer
+tile.computercraft:advanced_computer.name=Erweitertetr Computer
+tile.computercraft:drive.name=Diskettenlaufwerk
+tile.computercraft:printer.name=Drucker
+tile.computercraft:monitor.name=Monitor
+tile.computercraft:advanced_monitor.name=Erweiterter Monitor
+tile.computercraft:wireless_modem.name=WLAN-Modem
+tile.computercraft:wired_modem.name=LAN-Modem
+tile.computercraft:cable.name=LAN-Kabel
+tile.computercraft:command_computer.name=Befehlscomputer
+tile.computercraft:advanced_modem.name=Ender-Modem
+tile.computercraft:speaker.name=Lautsprecher
+
+tile.computercraft:turtle.name=Roboter
+tile.computercraft:turtle.upgraded.name=%s-Bot
+tile.computercraft:turtle.upgraded_twice.name=%s & %s-Bot
+tile.computercraft:advanced_turtle.name=Erweiterter Bot
+tile.computercraft:advanced_turtle.upgraded.name=Erweiterter %s-Bot
+tile.computercraft:advanced_turtle.upgraded_twice.name=Erweiterter %s & %s-Bot
+
+item.computercraft:disk.name=Diskette
+item.computercraft:treasure_disk.name=Rostige Diskette
+item.computercraft:page.name=Ausdruck
+item.computercraft:pages.name=Asdrücke
```

Comment:
This one could be "Gedruckte Blätter", or something similar. "Ausdruck" is ambiguous with an expression.